main/cmds_test.go

```diff
@@ -115,6 +116,27 @@ func Test_runCmd_pass(t *testing.T) {
 	require.Nil(t, err)
 }
 
+func Test_runCmd_withTestFile(t *testing.T) {
+	dir := filepath.Join(dataDir, "testing")
+	_, err := unzip(log.NewNopLogger(), "../testing/testing.zip", dataDir)
+
+	// print files in directory
+	files, err := ioutil.ReadDir(dir)
+	if err != nil {
```

Comment:
If you couldn't read the directory doesn't that imply an issue?  Does it make sense to proceed?

Reply:
fixed